app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -72,11 +72,25 @@ def healthcheck
     end
 
     def facilities
-      render(json: lighthouse_facilities_service.get_facilities(lighthouse_facilities_params))
+      lighthouse_facilities = lighthouse_facilities_service.get_facilities(lighthouse_facilities_params)
+
+      if params[:ves_active]
```

Comment:
⚠️ We'd need to modify the request in `vets-website` `src/applications/hca/components/FormFields/VaMedicalCenter.jsx:17` to pass in this parameter.

If we filtered unconditionally, it'd impact all requests to this endpoint - not sure which default is better?